include/libdnf5/conf/vars.hpp

```diff
@@ -101,7 +101,7 @@ struct Vars {
     /// @param name Name of the variable
     const Variable & get(const std::string & name) const { return variables.at(name); }
 
-    static std::unique_ptr<std::string> detect_release(const BaseWeakPtr & base, const std::string & install_root_path);
+    static std::string detect_release(const BaseWeakPtr & base, const std::string & install_root_path);
```

Comment:
We cannot do that, it would break API.
Either we could target this PR to the `dnf5-5.2.0.0` branch (which is a new major release with breaking changes) or we have to find another way. Perhaps we could simply ignore the function in go bindings for now.